plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -238,6 +241,18 @@ export function AboutCard(props: AboutCardProps) {
             >
               <EditIcon />
             </IconButton>
+            {sourceTemplateRef && templateRoute && (
+              <IconButton
+                component={Link}
+                title="Create something similar"
+                to={templateRoute({
+                  namespace: sourceTemplateRef.namespace,
+                  templateName: sourceTemplateRef.name,
+                })}
+              >
+                <CopyIcon />
```

Comment:
Thinking that using the `scaffolder` icon from the app icons might be a good option too?